### PR TITLE
updated benchmarks for dkg-metadata

### PR DESCRIPTION
**Pull request by salman01zp**
**Summary of changes**
Changes introduced in this pull request:
- added benchmarks for dkg-metadata pallet

### CODE REVIEW

**Comment by drewstone** [comment]
Looks like you need to run `cargo fmt` and ensure the weights are exported properly. Also fix up the licenses and any other things in the CI.

File: pallets/dkg-metadata/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+	let mut payload = Vec::new();
+	payload.extend_from_slice(round_id.to_be_bytes().as_ref());
+	payload.extend_from_slice(offender.clone().as_ref());
+
+	let signature = sr25519_sign(AURA,&pub_key,&payload).unwrap();
+
+	signature.encode()
+}
+
+fn mock_accoun_id<T:frame_system::Config>(pub_key:sr25519::Public)-><T as frame_system::Config>::AccountId{
```
Comment (drewstone) [inline pallets/dkg-metadata/src/benchmarking.rs:68]: Spelling is off `mock_accoun_id`

File: pallets/dkg-metadata/src/benchmarking.rs [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+						    utils::{sr25519,ecdsa}};
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+use sp_runtime::{Permill,key_types::AURA};
+use sp_io::crypto::{ecdsa_sign_prehashed,sr25519_sign,ecdsa_generate,sr25519_generate};
+use sp_runtime::traits::TrailingZeroInput;
+
+
+
+const MAX_AUTHORITIES: u32 = 10;
```
Comment (drewstone) [inline pallets/dkg-metadata/src/benchmarking.rs:34]: Let's use 100 as max authorities.

**Review by drewstone — CHANGES_REQUESTED** [review]
Formatting and spelling changes needed. Looks like you're missing spaces after `,` too. Likely, `cargo fmt` will fix that but if not please fix those up too.

File: pallets/dkg-metadata/src/benchmarking.rs [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+		where
+			T::DKGId: From<ecdsa::Public>,
+			T::AccountId : From<sr25519::Public>
+
+	}
+
+	set_signature_threshold {
+		// threshold should be less than total number of next authorities
+		let mut next_authorities:Vec<T::DKGId> = Vec::new();
+		for id in 1..=MAX_AUTHORITIES{
```
Comment (dharjeezy) [inline pallets/dkg-metadata/src/benchmarking.rs:94]: You can try and format here.

**Comment by dharjeezy** [comment]
LGTM
Try and fix the clippy and make CI green.

File: pallets/dkg-metadata/src/benchmarking.rs (hunk 1 above)
Comment (salman01zp) [reply pallets/dkg-metadata/src/benchmarking.rs]: updated.

**Review by dharjeezy — APPROVED** [review]

**Comment by drewstone** [comment]
Great work @salman01zp

**Pull request merged**